docs/_themes/clusterhq/layout_homepage.html

```diff
@@ -87,7 +87,7 @@
       </div>
     </div>
         
-    <h2 class="heading text-center doc-banner">See our <a href="https://docs.clusterhq.com/" title="Flocker documentation">documentation</a> to learn more</h2>
+    <h2 class="heading text-center doc-banner">See our <a href="{{ pathto('index') }}" title="Flocker documentation">documentation</a> to learn more</h2>
```

Comment:
might as well make this a complete sentence while you're here (missing a full stop at the end)